python/pyarrow/types.pxi

```diff
@@ -847,6 +847,25 @@ cdef timeunit_to_string(TimeUnit unit):
         return 'ns'
 
 
+FIXED_OFFSET_PREFIX = '+'
```

Comment:
@wesm suggestions?

Reply:
We probably want the offset to be encoded as `[+-]HH:MM`.
See https://github.com/apache/arrow/blob/master/format/Schema.fbs#L162-L166

Reply:
Done!